rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -18,20 +18,16 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from math import sqrt
 import torch
-import itertools
-import os
-
 from amd.rocal.pipeline import Pipeline
 import amd.rocal.fn as fn
 import amd.rocal.types as types
+import sys
```

Comment:
we need to port all the existing examples in image pipeline to tensor as well. Looks like bunch of them are not present here

Reply:
Yes Rajy, we have ported one classifictaion & one detection example. The other examples will be ported in the future PRs